### ISSUE TITLE
Can no longer access /home after upgrading to v4

### ISSUE THREAD

**Issue by nekename** [ajbura/cinny#1847]
### Describe the bug

After updating my [Cinny instance self-hosted on GitHub Pages](https://github.com/ninjadev64/cinny-web) and logging in, I am met with this error page. My homeserver works on app.cinny.in and in the desktop version of v4.0.3.

![image](https://github.com/user-attachments/assets/ade41cf2-d141-4196-9240-d6499dc8f552)

### Platform and versions

```shell
openSUSE Tumbleweed GNU+Linux
Mozilla Firefox 128.0
Cinny Web v4.0.3
```

**Comment by kfiven** [comment]
Hi, You need to setup redirects as explained in release notes.

**Comment by nekename** [comment]
> Hi, You need to setup redirects as explained in release notes.

How would I do that with GitHub Pages? Also, that should be added to the README, as currently it just says to extract the dist directory from the release to the webserver root.

**Comment by kfiven** [comment]
Can help you with netlify as that's our deployment type but https://blog.logrocket.com/deploying-react-apps-github-pages/ might help you with GitHub pages. Will update the readme with mentioning redirect setup requirement.

**Comment by martijndeb** [comment]
Please provide a working example for Caddy and other servers, just pointing to netlify.toml isn't the quality of docs that we expect on a breaking change. Reverted to the previous release

**Comment by kfiven** [comment]
You might want to check out https://caddyserver.com/docs/caddyfile/directives/redir

Do not expect a small team of two people to test redirect on every server out there and provide support for that.

Pointing to netlify meant, you know what redirect to set, but how to set is upto you because you deal with that server not us.

**Comment by martijndeb** [comment]
I don't, I do however expect up to date documentation on breaking changes. That should be part of writing the release notes. The documentation is behind reality now.

**Comment by kfiven** [comment]
As already mentioned above, will update readme on the requirement of setting redirect. On the doc issue, there is enough info because netlify.toml consit of exactly what need to set and release exactly mention that.

**Comment by nekename** [comment]
> Can help you with netlify as that's our deployment type but https://docs.github.com/en/contributing/writing-for-github-docs/configuring-redirects might help you with GitHub pages. Will update the readme with mentioning redirect setup requirement.

The link you sent is not for GitHub Pages, but instructions for contributors to GitHub's Docs.

**Comment by n3oney** [comment]
For anyone using Caddy, this seems to work for me:

```
        @index {
          not path /index.html
          not path /public/*
          not path /assets/*

          not path /config.json

          not path /manifest.json

          not path /pdf.worker.min.js
          not path /olm.wasm

          path /*
        }

        rewrite /*/olm.wasm /olm.wasm
        rewrite @index /index.html
```

**Comment by kfiven** [comment]
Got to know that you can also set https://github.com/cinnyapp/cinny/blob/dev/config.json#L35 this to true and you do not need to set redirects.

Closing as nothing more to be done from cinny side.

**Comment by nekename** [comment]
> Got to know that you can also set https://github.com/cinnyapp/cinny/blob/dev/config.json#L35 this to true and you do not need to set redirects.

Too late, you've already lost a user.

**Comment by paboum** [comment]
I had to symlink my `/var/www/cinny/` to `/var/www/cinny/home` for some reason. I don't care about redirects, this should work straight ahead after unzipping, just as element, schildi, or other clients do. While at it, I also tried running this without a subdomain only to learn another pathing issue occurs and I need to rebuild the whole stuff with a path changed, which obviously fails on small vps machines.

This is the last time I'm contributing to this project with my mindful insight, until this issue becomes properly resolved.

> Too late, you've already lost a user.

Ah yes, they also censor criticism, nice.
